corehq/apps/es/users.py

```diff
@@ -150,7 +145,3 @@ def user_ids_at_locations_and_descendants(location_ids):
 
 def user_ids_at_locations(location_ids):
     return UserES().users_at_locations(location_ids).exclude_source().run().hits
-
```

Comment:
Noah, 
i have used the method `user_ids_at_accessible_locations` and hence `users_at_accessible_locations` in UserES in couple of PRs and are now on staging. Can you put these back so they are not removed on staging, unless there is a strong reason to remove them or if there is an alternate?